corona-warn-app/cwa-app-android
# Duration selection screen (EXPOSUREAPP-8949)

Duration selection screen for event warning.

<img width="220" alt="Screenshot 2021-08-18 at 16 31 30" src="https://user-images.githubusercontent.com/64849422/129907505-4313bfc5-4ea1-4bad-b9df-b674c81e27ac.png">

How to test:
1) Go to event organiser screen and create several events (in past, today, future, two and more days length)
2) Go to warning menu -> info screen -> event selection screen
3) Check each event created

ToDo:
Navigation events and navigation to the next screen would be handle in another PR.

To reviewer:
I added some comments from technical acceptance criteria for better understanding and readability.
